advocacy_docs/partner_docs/VeritasGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for Veritas and NetBackup for PostgreSQL'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      |  Veritas     |
+| **Partner Product**   |   NetBackup for PostgreSQL    |
+| **Web Site**      |   https://www.veritas.com/   |
+| **Version & Platform**   |   NetBackup for PostgreSQL 9.1: Linux, Windows     |
+| **Product Description**      |  Database backup/restore tool     |
```

Comment:
The product description should mention the company name and summarize the overall purpose of the integration. It should answer the question: "What does this integration do?" in a single sentence.